client/js/contextMenu.js

```diff
@@ -0,0 +1,85 @@
+"use strict";
+const $ = require("jquery");
+const templates = require("../views");
+let contextMenu, contextMenuContainer;
+
+module.exports = class ContextMenu {
+	constructor(contextMenuItems, contextMenuActions, selectedElement, event) {
+		this.contextMenuItems = contextMenuItems;
+		this.selectedElement = selectedElement;
+		this.event = event;
+
+		contextMenuContainer = $("#context-menu-container");
+		contextMenu = $("#context-menu");
+
+		contextMenuContainer.on("click contextmenu", function() {
```

Comment:
You're going to be binding these events multiple times, as far as I understand it, as the container is created once unlike this constructor.

Reply:
Good point, I've put this with the other ones in `lounge.js`. It's not ideal, but at least this is a very small amount of code there, which won't grow, unlike the rest of the context menu code.